Kudu.Services.Web/Default.cshtml

```diff
@@ -73,26 +73,29 @@
             <a href="api/deployments">Deployments</a>
         </li>
         <li>
-            <a href="api/vfs">Files</a>
-        </li>
-        <li>
-            <a href="api/processes">Processes and mini-dumps</a>
-        </li>
-        <li>
-            <a href="api/diagnostics/runtime">Runtime versions</a>
-        </li>
-        <li>
-            Site Extensions: <a href="api/siteextensions">installed</a> | <a href="api/extensionfeed">feed</a>
-        </li>
-        <li>
             <a href="api/scm/info">Source control info</a>
```

Comment:
The ordering of 'Source control info' is different from before. Not sure if that's intentional.